### PR TITLE
Fix nesting of shared directory for checkpoint

### PR DESCRIPTION
**Pull request by dwhswenson**
Per docs on `MultistateReporter`, ["this should NOT be a full path, and instead just a filename"](https://github.com/choderalab/openmmtools/blob/abd4011f1c756722ab8671097335c0e895f5741f/openmmtools/multistate/multistatereporter.py#L93).

Going to add tests to `test_openmm_run_engine` to check this -- that is the appropriate "moist" test, right?

### CODE REVIEW

**Comment by IAlibay** [comment]
> that is the appropriate "moist" test, right?

Yes

File: openfe/tests/protocols/test_openmm_equil_rfe_protocols.py [hunk 1]
```diff
@@ -811,7 +811,7 @@ def set_openmm_threads_1():
             os.environ['OPENMM_CPU_THREADS'] = previous
 
 
-@pytest.mark.slow
+# @pytest.mark.slow
```
Comment (dwhswenson) [inline openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:814]: Revert this before merge

**Comment by dwhswenson** [comment]
Ready for review. Turned on the slow test for this, turn it back off before merge.

File: openfe/tests/protocols/test_openmm_equil_rfe_protocols.py (hunk 1 above)
Comment (dwhswenson) [reply openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:814]:
```suggestion
@pytest.mark.slow
```

**Review by richardjgowers — APPROVED** [review]
checked locally, works

**Pull request merged**